### PR TITLE
Add optipng compression of res/*.png

### PR DESCRIPTION
**Pull request by sigvef**
image

### CODE REVIEW

File: nin/backend/compile.js [hunk 1]
```diff
@@ -43,12 +47,34 @@ function res(projectPath, callback) {
     root += '/';
     root = root.replace(/\/\//g, '/');
 
-    const file = fs.readFileSync(root + stat.name);
+    let file = fs.readFileSync(root + stat.name);
```
Comment (aleksanb) [inline nin/backend/compile.js]: Nitpick: file doesn't seem to be redeclared in scope?

**Comment by aleksanb** [comment]
Could you add your changes to the yarn.lock as well?

File: nin/backend/compile.js (hunk 1 above)
Comment (sigvef) [inline nin/backend/compile.js]: I changed it for a first implementation that redeclared `file`, but then I forgot to change it back when I changed to a different implementation that didn't redeclare it. Fixed.

**Comment by sigvef** [comment]
ping

**Pull request merged**